google/fingerprinters/web/scripts/updater/community/solr/app/docker-compose.yaml

```
@@ -0,0 +1,13 @@
+version: '3'
+services:
+  solr:
+    image: solr:${SOLR_VERSION}
+    ports:
+     - "8983:8983"
+    volumes:
+      - data:/var/solr
+    command:
+      - solr-precreate
+      - gettingstarted
+volumes:
+  data:
```

```suggestion
    command:
      - solr-precreate
      - gettingstarted
```
Please don't use volumes as we don't want to persist data